packages/shared/src/interface/external/athenahealth/problem.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+export const problemCreateResponseSchema = z.object({
+  success: z.boolean(),
+  errormessage: z.string().optional(),
+  problemid: z
+    .number()
+    .transform(val => `${val}`)
```

Comment:
I thought we agreed to just parse as `string`? Or is there the need to parse it to number and fail if they add something that's not a number to it?

Reply:
switched to coerce